### PR TITLE
Dockerize

### PR DESCRIPTION
**Pull request by cmonez**
Dockerized photos service :) @andyoe0102 @samasastudio @krisdo

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    ports:
+      - '3001:3001'
+    depends_on:
+      - mongodb
+    volumes:
+      - ./:/photoapp/
+      - /photoapp/node_modules
+      - ./wait-for-it.sh:/photoapp/wait-for-it.sh
+    command: bash -c "/photoapp/wait-for-it.sh --timeout=0 mongodb:27017 && npm run docker"
+  mongodb:
```
Comment (samasastudio) [inline docker-compose.yml:16]: Did you try commenting out line 14 to see if it makes a difference?  I'm curious...

File: database/connection.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 const mongoose = require('mongoose');
 
-const mongoUri = 'mongodb://localhost/fec-photos';
+const mongoUri = 'mongodb://mongodb:27017/fec-photos';
 const db = mongoose.connect(mongoUri, {
```
Comment (samasastudio) [inline database/connection.js:4]: Was this the change that fixed everything?  So it tried to connect to the db container?

File: docker-compose.yml (hunk 1 above)
Comment (cmonez) [reply docker-compose.yml:16]: I havent. But i will and will let u know.

File: database/connection.js (hunk 2 above)
Comment (cmonez) [reply database/connection.js:4]: Yes! @krisdo found the mistake in like 3 seconds.

**Pull request merged**